### ISSUE TITLE
CLI creates OAuth config in awsconfiguration.json when not needed

### ISSUE THREAD

**Issue by rohandubal** [aws-amplify/amplify-cli#1470]
**Describe the bug**
CLI creates OAuth config in awsconfiguration.json even when OAuth is not configured or enabled.

**To Reproduce**
Enable default auth for iOS

**Expected behavior**
No OAuth config

**Comment by kaustavghosh06** [comment]
Can you past the contents of your awsconfiguration.json and amplify/backend/amplify-meta.json file?

**Comment by rohandubal** [comment]
Amplify meta

```
{
	"providers": {
		"awscloudformation": {
			"AuthRoleName": "gh1248-dev-xxx-authRole",
			"UnauthRoleArn": "arn:aws:iam::xxxxxx:role/gh1248-dev-20190516140121-unauthRole",
			"AuthRoleArn": "arn:aws:iam::xxxx:role/gh1248-dev-20190516140121-authRole",
			"Region": "us-west-2",
			"DeploymentBucketName": "gh1248-dev-20190516140121-deployment",
			"UnauthRoleName": "gh1248-dev-20190516140121-unauthRole",
			"StackName": "gh1248-dev-20190516140121",
			"StackId": "arn:aws:cloudformation:us-west-2:xxxxx:stack/gh1248-dev-20190516140121/c23504b0-781d-11e9-8cfa-02e409a5553a"
		}
	},
	"auth": {
		"cognito10fc382d": {
			"service": "Cognito",
			"providerPlugin": "awscloudformation",
			"providerMetadata": {
				"s3TemplateURL": "https://s3.amazonaws.com/gh1248-dev-xxx-deployment/amplify-cfn-templates/auth/cognito10fc382d-cloudformation-template.yml",
				"logicalId": "authcognito10fc382d"
			},
			"lastPushTimeStamp": "2019-05-16T21:07:09.957Z",
			"output": {
				"AppClientSecret": "xxx",
				"UserPoolId": "us-west-xxxxx",
				"AppClientIDWeb": "xxxxx",
				"AppClientID": "xxxxx",
				"IdentityPoolId": "us-west-2:xx-4d55-4535-xx-xxx",
				"IdentityPoolName": "xxxx",
				"UserPoolName": "xxx"
			},
			"lastPushDirHash": "xxx+xx="
		}
	}
}
```


awsconfiguration.json

```
{
    "UserAgent": "aws-amplify/cli",
    "Version": "0.1.0",
    "IdentityManager": {
        "Default": {}
    },
    "CredentialsProvider": {
        "CognitoIdentity": {
            "Default": {
                "PoolId": "us-west-2:xxxx-4d55-xxxx-xx-xxx",
                "Region": "us-west-2"
            }
        }
    },
    "CognitoUserPool": {
        "Default": {
            "PoolId": "us-west-xxxx",
            "AppClientId": "xxxxx",
            "AppClientSecret": "xxxx",
            "Region": "us-west-2"
        }
    },
    "Auth": {
        "Default": {
            "OAuth": {
                "AppClientId": "xxxx",
                "AppClientSecret": "xxxx"
            }
        }
    }
}
```

**Comment by kaustavghosh06** [comment]
Hey! Submitted a fix in #1472
